hera_pspec/plot.py

```diff
@@ -494,3 +496,176 @@ def delay_waterfall(uvp, blpairs, spw, pol, component='real', average_blpairs=Fa
     if new_plot:
         return fig
     
+def delay_wedge(uvps, blpairs, fold=False, cosmo=True, center_line=True, horizon_lines=True, cosmo_cbar=True, suptitle=''):
```

Comment:
This function should optionally take an `ax=None` kwarg, if the user wants to instantiate their own figure and subplots themselves

Reply:
Additionally, you should request a `spw` and a `pol` as *arguments*. See other plotting functions for example